src/components/App/App.js

```diff
@@ -24,21 +26,45 @@ class App extends Component {
     .finally(() => this.setState({ isLoading: false }))
   }
 
+
+  searchByTitle = (searchCriteria) => {
+
+   let movieList
+
+    if(!isNaN(searchCriteria)) {
```

Comment:
Let's look into whether we can make this _not_ a double negative if we have time.